README.md

```diff
@@ -44,56 +47,58 @@ If you're running Windows, you may need to add the following lines to your `C:\W
 
 On Linux, anything with a `.localhost` domain is automatically mapped to 127.0.0.1 so you don't need to edit your `/etc/hosts` file.
 
-### Optional setup for debugging
-
-If you want to test out Honeycomb traces, you will need to set the `HONEYCOMB_API_KEY` environment variable in
-the `deployment/dev/local.env` file.
-You can get the key from [here](https://ui.honeycomb.io/sil-language-forge/environments/test/api_keys)
-
-#### git note
-this repo uses a ignore revs file. To configure this repo to use it run this command. It should be executed as part of `task setup`
-
-```bash
-git config blame.ignoreRevsFile .git-blame-ignore-revs
-```
-
-#### for local dev also:
- * node version 18+
- * dotnet 7 sdk
-
-### Docker workflow
+### Kubernetes workflow
 
 ```bash
 task up
 ```
-The full app will be running on http://localhost after everything starts.
+The full app will be running at http://localhost after everything starts.
 There are some additional urls below to access specific parts of the system.
 
 ### Local workflow
-you can run the front and back end together (console output will be mixed) or in different terminals. All necessary infrastructure (`task infra`) will be started automatically.
 
-frontend
-```bash
-task ui
-```
-backend
-```bash
-task api
-```
-both
-```bash
-task dev
-```
+#### Prerequisites
+- The SvelteKit UI requires: node v20+
+- The .NET API requires: dotnet sdk v8+
+
+#### Running the project
+
+There are various ways to run the project. Here are a few suggestions:
+
+**For developing the .NET API**
+- `task infra-up` starts all necessary infrastructure in k8s
+- `task api:only` starts the api locally
+
+**For developing the SvelteKit UI**
+
+1) In two seperate consoles:
+- `task backend-up` starts all necessary infrastructure + the .NET API in k8s
+- `task ui:only` starts the ui locally
+1) In a shared console:
+- `task ui-dev`
 
-pnpm should be installed automatically using nodejs corepack, if not you can run `corepack enable` to install it.
+The SvelteKit UI will be available at http://localhost:3000.
+
+> [!IMPORTANT]
+> The SvelteKit UI is always available in k8s at http://localhost, but will not be reliable unless the entire project is started with `task up`.
+> 
+**For developing the .NET API and the SvelteKit UI**
+- `task infra-up` starts all necessary infrastructure in k8s
+- `task api:only` starts the api locally
+- `task ui:only` starts the ui locally
+
+**If the k8s deployments are already running**
+
+- `infra-forward` forwards the infrastructure ports for the API
+- `backend-forward` forwards the infrastructure + backend ports for the UI
 
 ---
-### Helpful urls
-* http://localhost - k8s ingress
-* http://localhost:3000 - sveltekit frontend
-* http://localhost:5158/api/swagger - swagger docs for the api
-* http://localhost:5158/api/graphql/ui - graphiql UI
-* http://localhost:5158/api/graphql - graphiql endpoint
+### Project urls
+* http://localhost - k8s ingress / SvelteKit UI
```

Comment:
if you're going to mention sveltekit here then the api should also be mentioned. That said everything goes to ingress which routes it to the correct location